pyvista/plotting/lights.py

```diff
@@ -93,22 +127,36 @@ def __init__(self, position=None, focal_point=None, color=None, light_type='scen
             type_normalized = light_type.replace(' ', '').lower()
             mapping = {'headlight': LightType.HEADLIGHT,
                        'cameralight': LightType.CAMERA_LIGHT,
-                       'scenelight': LightType.SCENE_LIGHT,
-                      }
+                       'scenelight': LightType.SCENE_LIGHT
+            }
```

Comment:
Then either go with
```py
mapping = {'headlight': LightType.HEADLIGHT,
           'cameralight': LightType.CAMERA_LIGHT,
           'scenelight': LightType.SCENE_LIGHT}
```
or
```py
mapping = {
    'headlight': LightType.HEADLIGHT,
    'cameralight': LightType.CAMERA_LIGHT,
    'scenelight': LightType.SCENE_LIGHT
}   
```

I like keeping the trailing comma on the last item and the closing bracket on a separate line, because in this case adding new items would only show diffs with the new additions. But of course it's unlikely that we'll get new light types any time soon.

Reply:
Went with the latter as it's unlikely we'll get new light types.